docs/advanced-chart-features/marker-clusters.md

```diff
@@ -0,0 +1,117 @@
+Marker clusters
+===
+
+## Introduction
+Marker clusters is the concept of sampling the data values into larger blocks in order to ease readability and increase performance. Is a simple solution to display a large number of markers on a map or a chart. The number on a cluster shows how many markers it contains. As you zoom into the map the points will start to show and the cluster will contain less markers.
+
+The `marker-clusters` module supports `mappoint` and `scatter` series types.
+
+**Maps demo**
+
+<iframe style="width: 100%; height: 450px; border: none;" src=https://www.highcharts.com/samples/embed/maps/marker-clusters/europe allow="fullscreen"></iframe>
+
+**Scatter demo**
+
+<iframe style="width: 100%; height: 450px; border: none;" src=https://www.highcharts.com/samples/embed/highcharts/marker-clusters/basic allow="fullscreen"></iframe>
+
+Installation
+------------
+
+Requires `modules/marker-clusters.js`. To display marker clusters, set `series.cluster.enabled` to `true`.
+
+Configuration Options
+-------------
+
+The main marker clusters configuration is set in the `cluster property in the series options.`
+
+|Option|Description|
```

Comment:
Don't repeat the API options. This will be a maintenance problem if we change the API and forget to change here. Instead, give the highlights and explain the main features in the docs article, and provide a link to the detailed API.